doc/source/reference/routines.char.rst

```diff
@@ -90,3 +90,4 @@ Convenience class
    array
    asarray
    chararray
+   compare_chararrays
```

Comment:
Seems to me this belongs under the "Comparison" heading

Reply:
Oh! I'm making a checklist of the items that I can fix, so I will review this one as well

Reply:
You're right. I'll move it